### DISCUSSION TITLE
part-ii/2019-98/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#101] [Question comments]
Part II 2019 Number Fields: Paper 1, Section II, 20G 

https://questions.tripos.org/part-ii/2019-98/

**Comment by artie2000** [comment]
I got unit group $\{\pm(1+\sqrt{2})^n\mid n\in\mathbb{Z}\}$,
<details>
<summary>and generator</summary>

$-7-5\sqrt{2}$

</details>

**Reply by callumhobbis** [reply]
Me too!
